integration-cli/docker_cli_search_test.go

```diff
@@ -63,6 +63,16 @@ func (s *DockerSuite) TestSearchCmdOptions(c *check.C) {
 		c.Fatalf("failed to search on the central registry: %s, %v", outSearchCmd, err)
 	}
 
+	searchCmdNotrunc := exec.Command(dockerBinary, "search", "--no-trunc=true", "busybox")
+	outSearchCmdNotrunc, exitCode, err := runCommandWithOutput(searchCmdNotrunc)
+	if err != nil || exitCode != 0 {
```

Comment:
no big deal but exitCode should be always not equal to zero if an error occurred